packages/cli/src/voting/getRewardsByRoundId.js

```diff
@@ -86,7 +86,7 @@ const getRewardsByRound = async (web3, votingContract, account) => {
     });
   });
   roundIds = roundIds.sort((a, b) => {
-    return parseInt(a.id) - parseInt(a.id);
+    return parseInt(a.id) - parseInt(b.id);
```

Comment:
This was a bad bug, how did eslint catch it??

Reply:
unused variable haha